hazelcast-client/src/main/java/com/hazelcast/client/impl/clientside/HazelcastClientInstanceImpl.java

```diff
@@ -217,6 +216,7 @@
     public HazelcastClientInstanceImpl(ClientConfig config,
                                        ClientConnectionManagerFactory clientConnectionManagerFactory,
                                        AddressProvider externalAddressProvider) {
+        probeRegistry = new ProbeRegistryImpl();
```

Comment:
If you don't need to pass any arguments, I would just initialize the field directly and keep the constructor clean(er).

Reply:
We are within a `try-catch` here. Not sure what to do.